vnext/ReactUWP/Views/ViewControl.h

```diff
@@ -44,30 +62,43 @@ struct ViewControl : winrt::Windows::UI::Xaml::Controls::ContentControlT<ViewCon
 
 private:
   AccessibilityRoles m_accessibilityRole = AccessibilityRoles::None;
+  bool m_accessibilityStates[AccessibilityStates::CountStates] = { 0 };
   AccessibilityInvokeEventHandler m_accessibilityInvokeHandler = nullptr;
 };
 
-}} // namespace react::uwp
-
-
-#include "cppwinrt/react.uwp.DynamicAutomationPeer.g.h"
-namespace winrt::react::uwp::implementation
-{
 //
 // DynamicAutomationPeer refers to the owner ViewControl to determine what type control
 // it appears to be for accessibility tools
 //
-struct DynamicAutomationPeer : DynamicAutomationPeerT<DynamicAutomationPeer>
+struct DynamicAutomationPeer : winrt::FrameworkElementAutomationPeerT<DynamicAutomationPeer, winrt::IInvokeProvider, winrt::ISelectionProvider, winrt::ISelectionItemProvider>
```

Comment:
>FrameworkElementAutomationPeerT [](start = 38, length = 31)

great, I had trouble trying to define this myself before and the idl route had made it easier, glad its simplified

Reply:
Unfortunately I started running into trouble with how UIA interacts with the peers (causing AVs), and went back to the idl method for this particular class. I'm not sure it's the cause, but I'd rather not mess with the whole inner/outer pointer winrt stuff.